### PR TITLE
Shows spinner during API request load.

### PR DESCRIPTION
**Pull request by ashfurrow**
This PR shows a spinner while the necessary API requests to display the artwork actions view (bid button, contact gallery button, countdown timer, etc) are fetched. 

The bug was incorrectly identified as a regression. We saw the beta behave as described in the ticket, and saw the App Store version working fine a few minutes later. During that time, I believe a staging blip corrected itself. I could no longer produce the lag on the beta _but_ sometimes API requests take time, so we should still show the spinner while they're in flight.

Fixes [Purchase-292](https://artsyproduct.atlassian.net/browse/PURCHASE-292).

### CODE REVIEW

File: Artsy/Views/Artwork/ARArtworkActionsView.m [hunk 1]
```diff
@@ -63,13 +63,25 @@ - (void)setDelegate:(id<ARArtworkActionsViewDelegate, ARArtworkActionsViewButton
...
 
+- (void)didMoveToWindow
+{
+    [super didMoveToWindow];
+
+    // New ARArtworkViews are instantiated and loaded while not in a view hierarchy. The call to fadeInAnimated: in
+    // `showSpinner` will have no effect until the view is in a hierarchy, so we check below.
+    if (self.superview) {
+        [self.spinner fadeInAnimated:ARPerformWorkAsynchronously];
+    }
```
Comment (orta) [inline Artsy/Views/Artwork/ARArtworkActionsView.m]: nice trick 👍

**Comment by ashfurrow** [comment]
Hold off on merging for now, this needs a slight tweak for iPhones so we don't show two spinners at once:

screen shot 2018-07-18 at 4 10 29 pm

**Comment by ashfurrow** [comment]
This is done and ready for review 👍

**Comment by sweir27** [comment]
Looks good! Feel free to resolve the changelog and merge on green. 😄

**Pull request merged**